runtime/src/lib.rs

```diff
@@ -190,36 +233,38 @@ pub const DAYS: BlockNumber = HOURS * 24;
 pub const CONTRACTS_DEBUG_OUTPUT: bool = true;
 
 const fn deposit(items: u32, bytes: u32) -> Balance {
-	items as Balance * 15 * UNIT + (bytes as Balance) * 6 * UNIT
+	items as Balance * 15 * UNITS + (bytes as Balance) * 6 * UNITS
 }
 
 // 1 in 4 blocks (on average, not counting collisions) will be primary babe blocks.
 pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
 
-// Unit = the base number of indivisible units for balances
-pub const UNIT: Balance = 1_000_000_000_000;
-pub const MILLIUNIT: Balance = 1_000_000_000;
-pub const MICROUNIT: Balance = 1_000_000;
+pub const UNITS: Balance = 10_000_000_000;
+pub const DOLLARS: Balance = UNITS;
+pub const CENTS: Balance = UNITS / 100;
+pub const MILLICENTS: Balance = CENTS / 1_000;
+
+/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9020).
```

Comment:
What does "v9020" refer to?

Reply:
We can remove that. It is the spec version of the relay chain.